### DISCUSSION TITLE
How to disable all LS?

### DISCUSSION THREAD

**Discussion by GildedHonour** [zed-industries/zed#22507] [Feature Requests]
How to disable all Language Servers for all the languages at once?

Optionally, disable all the *runtime* analyzers, pre-compilers, all kinds of the suggestors, and all the shit too.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
